plugins/lookup/output.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+name: output
+author: Polona Mihalič (@PolonaM)
+version_added: 1.0.0
+short_description: Read state file outputs.
+description:
+    - This lookup returns all outputs or selected output in state file.
+options:
+  _terms:
+    description:
+    - Names of the outputs to return.
+    - If value is set to C(all), all outputs will be returned.
+    required: True
+  project_path:
+    description:
+      - The path to the root of the Terraform directory with the terraform.tfstate file.
+    type: path
+  state_file:
+    description:
+      - Absolute path to an existing Terraform state file whose outputs will be listed.
+      - If this is not specified, the default C(terraform.tfstate) in the directory I(project_path) will be used.
+    type: path
+"""
+
+EXAMPLES = """
+- name: get selected output from terraform.tfstate
+  ansible.builtin.debug:
+    msg: "{{ lookup('cloud.terraform.output', 'my_output1', 'my_output2', project_path='path/to/project/dir/') }}"
+
+- name: get all outputs from custom state file
+  ansible.builtin.debug:
+    msg: "{{ lookup('cloud.terraform.output', 'all', state_file='path/to/custom/state/file') }}"
+"""
+
+RETURN = """
+_value:
+  description:
+    - Requested outputs.
+  type: list
+  elements: dict
```

Comment:
Always a list of dicts? Not a dict or a bare value?

Reply:
You are probably right, and returning only output value if output is specified, makes more sense. So I updated it to return a list of values (and not dict) in case of specifying which outputs one wants to return. And yes, always a list of something:)